File: backend/demo/src/main/java/com/example/demo/models/Ronda.java
package com.example.demo.models;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Ronda {
    private Long idRonda;
    private Long idTorneo;
    private Integer numeroRonda;
    private Long idZonaAmbiental;
}

File: backend/src/main/kotlin/com/example/demo/models/Ronda.kt
package com.example.demo.models

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "rondas")
data class Ronda(
    @Id
    val id: String? = null,
    val torneoId: String,
    val numeroRonda: Int,
    val estado: String = "PENDIENTE"
)